tests/remote_js/httpHandler.js

```diff
@@ -27,26 +27,9 @@ class HTTP {
 			url = url.replace(localIPRegex, 'localhost');
 		}
 
-		let success = false;
-		let attempts = 3;
-		let tried = 0;
-		let response;
-		while (!success && tried < attempts) {
-			try {
-				response = await fetch(url, options);
-				success = true;
-			}
-			catch (error) {
-				if (error.name === 'FetchError') {
-					console.log('Request aborted. Wait for 2 seconds and retry...');
-					await new Promise(r => setTimeout(r, 2000));
-					tried += 1;
-				}
-			}
-		}
-		if (!success) {
-			throw new Error(`${method} to ${url} did not succeed after ${attempts} attempts.`);
-		}
+		await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
You should add a comment with a link to the GitHub issue.

Reply:
I just meant to comment the line so we know why this is here, not to add a comment on GitHub.

Reply:
Got it! My bad

Reply:
I mean literally add a comment to the source code above this line and commit it. Nothing to do with our GitHub. There just shouldn’t be random unexplained lines in the source code that will stay there forever because we don’t know why they’re there.

Reply:
Understood. Sorry about that